src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -589,24 +591,27 @@ public boolean updateUserRolesInIdam(CaseWorkersProfileCreationRequest cwrProfil
                 validationServiceFacade.logFailures(UP_FAILURE_ROLES, cwrProfileRequest.getRowId());
                 return false;
             }
-
-            Set<String> mappedRoles = getUserRolesByRoleId(cwrProfileRequest);
             UserProfileResponse userProfileResponse = (UserProfileResponse) requireNonNull(responseEntity.getBody());
```

Comment:
requireNonNull will throw nullpointerexception and will return 500 , is this being handled in exception mapper?

Reply:
It is existing code, so should probably being handled already